app/views/posts/index.html.erb

```diff
@@ -6,5 +6,5 @@
   <% end %>
 </div>
```

Comment:
- [ ] In the _comments partial, you should display both the author name and text. At the moment, your code is displaying just the text
`<p><%= comment.author.name %>: <%= comment.text %></p>`

Reply:
well resolved! ✔️